docs/topics_5/rn-resolved-issues.adoc

```diff
@@ -7,23 +7,23 @@ At the time of the release the following resolved issues have been identified as
 
 For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{ProductVersion} resolved issues] in Jira.
 
-.Resolved issues
+.Major resolved issues
 [cols="25%,75%",options="header"]
 |====
 |ID
 |Summary
 
 |link:https://issues.redhat.com/browse/WINDUP-2941[WINDUP-2941]
-|MTA reports load slowly, do not include Target Runtime labels on Application list, and do not have working feedback controls.
+|MTA reports load slowly, do not include Target Runtime labels on the *Application List* screen, and do not have working feedback controls.
 
 |link:https://issues.redhat.com/browse/WINDUP-2333[WINDUP-2333]
-|The MTA web console installed on OpenShift by means of a template cannot access the reports using HTTPS.
+|The {WebName} installed on OpenShift 4.4 and earlier versions cannot access the generated reports over an HTTPS connection.
 
 |link:https://issues.redhat.com/browse/WINDUP-2801[WINDUP-2801]
-|The MTA web console installed on OpenShift by means of the MTA Operator cannot access the reports using HTTPS.
+|The {WebName} installed on OpenShift 4.5 and later versions cannot access the generated reports over an HTTPS connection.
 
 |link:https://issues.redhat.com/browse/WINDUP-2910[WINDUP-2910]
-|"Unparseable Class File" errors are generated when MTA analyzes applications built with JDK 11.
+|Analysis of applications built with JDK 11 generates `"Unparseable Class File"` errors.
```

Comment:
You should remove the quotes. Just the backtick markup should be around the error message.